### PR TITLE
refactor: sort mixins and accesswideners

### PR DESCRIPTION
**Pull request by magicus**

### CODE REVIEW

File: common/src/main/resources/wynntils.accessWidener [hunk 1]
```diff
@@ -1,44 +1,44 @@
-accessWidener v2 named
```
Comment (kristofbolyai) [inline common/src/main/resources/wynntils.accessWidener]: why newline?

File: common/src/main/resources/wynntils.mixins.json [hunk 2]
```diff
@@ -2,22 +2,23 @@
   "client": [
     "AbstractContainerScreenMixin",
```
Comment (kristofbolyai) [inline common/src/main/resources/wynntils.mixins.json:3]: I thought these are sorted by spotless... At least, they used to be

File: common/src/main/resources/wynntils.accessWidener (hunk 1 above)
Comment (magicus) [reply common/src/main/resources/wynntils.accessWidener]: my bad, i did not check properly after sorting.

File: common/src/main/resources/wynntils.mixins.json [hunk 3]
```diff
@@ -42,7 +43,6 @@
     "accessors.MinecraftAccessor",
     "accessors.OptionsAccessor",
     "invokers.ChatScreenInvoker",
```
Comment (kristofbolyai) [inline common/src/main/resources/wynntils.mixins.json]:
```suggestion
    "invokers.ChatScreenInvoker"
```

File: common/src/main/resources/wynntils.mixins.json (hunk 2 above)
Comment (magicus) [reply common/src/main/resources/wynntils.mixins.json:3]: They don't seem to be properly sorted right now anyway, and spotless has not complained. However, I see that the IntelliJ sort I used seem to be case-sensitive and that the old sort was not:

```
-    "ClientPacketListenerMixin",
     "ClientPackSourceMixin",
+    "ClientPacketListenerMixin",
```

I don't know which is "right", or at least "best". I think it is convenient to use "sort lines" in IntelliJ, but non-case sensitive sorting seems saner, I think.

**Comment by magicus** [comment]
Argh. I though this was more a less a no-brainer, and I got a bit too quick about it. Is the case-sensitive sort order a problem? And can we get Spotless to sort json files for us? That'd be great.

**Comment by kristofbolyai** [comment]
> Argh. I though this was more a less a no-brainer, and I got a bit too quick about it. Is the case-sensitive sort order a problem? And can we get Spotless to sort json files for us? That'd be great.

I think we can. I will look into it later.

**Review by kristofbolyai — DISMISSED** [review]

**Comment by P0keDev** [comment]
i'm with bolyai in that i could've sworn that we had a custom spotless task for json ordering at one point

IMO case insensitive ordering makes more sense

**Comment by magicus** [comment]
@P0keDev We have, but of course it cannot reorder elements in an array. There is no way for it to know that it would be safe and okay in this particular case.

**Pull request merged**